mappings/net/minecraft/block/Blocks.mapping

```diff
@@ -167,6 +173,10 @@ CLASS net/minecraft/class_1752 net/minecraft/block/Blocks
 	FIELD field_7424 PLANKS Lnet/minecraft/class_197;
 	FIELD field_7425 SAPLING Lnet/minecraft/class_197;
 	FIELD field_7426 BEDROCK Lnet/minecraft/class_197;
+	FIELD field_7427 FLOW_WATER Lnet/minecraft/class_153;
+	FIELD field_7428 WATER_BLOCK Lnet/minecraft/class_197;
+	FIELD field_7429 FLOW_LAVA Lnet/minecraft/class_153;
```

Comment:
FLOWINF_LAVA

Reply:
now why would that be better

Reply:
flowing lava
better because upstream uses it